<ui version="4.0" >
 <class>ImportUserDatasetsClass</class>
 <widget class="QDialog" name="ImportUserDatasetsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>446</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>446</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Import datasets</string>
  </property>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>import</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_3" >
     <property name="geometry" >
      <rect>
       <x>290</x>
       <y>250</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text" >
      <string>Import selected</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>80</y>
       <width>271</width>
       <height>231</height>
      </rect>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Folder for datasets</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>C:/datasets/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Available datasets</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4" >
     <property name="geometry" >
      <rect>
       <x>260</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>Check available files</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox" >
     <property name="geometry" >
      <rect>
       <x>290</x>
       <y>140</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title" >
      <string>Log following errors</string>
     </property>
     <widget class="QCheckBox" name="checkBox_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>no excluded field</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>no accepted field</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>incomplete comp.</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
      <property name="tristate" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>no compartment</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
      <property name="tristate" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_5" >
     <property name="geometry" >
      <rect>
       <x>290</x>
       <y>120</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>auto run fix algorithms</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5" >
     <property name="geometry" >
      <rect>
       <x>290</x>
       <y>280</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text" >
      <string>Import all</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>download</string>
    </attribute>
    <widget class="QPushButton" name="pushButton" >
     <property name="geometry" >
      <rect>
       <x>290</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>download</string>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>URL for datasets</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>http://www.charite.de/bioinf/strap/php/metannogen/datasets/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>available datasets</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2" >
     <property name="geometry" >
      <rect>
       <x>260</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>check available datasets</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>80</y>
       <width>271</width>
       <height>231</height>
      </rect>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
